api/triggers.yaml

```diff
@@ -11,8 +11,11 @@ components:
         triggerType:
           type: string
           enum: ["http", "grpc", "traceid"]
+        # TODO: remove after test migration
         http:
           $ref: "./http.yaml#/components/schemas/HTTPRequest"
+        httpRequest:
```

Comment:
the openapi specs are being updated to match the keys on the file formats. They need to match to avoid the need for "transcoding" between the formats, and it's more "stable" to update the openapi instead of making users change their files